.github/workflows/check-rebaseable.yml

```diff
@@ -16,21 +16,19 @@ jobs:
   check-rebaseable:
     runs-on: ubuntu-latest
     env:
-      GH_TOKEN: ${{ secrets.DETERMINED_TOKEN }}
-      GITHUB_TOKEN: ${{ secrets.DETERMINED_TOKEN }}
       EE_REPO: determined-ai/determined-ee
+      OSS_REPO: determined-ai/determined
     steps:
-      - name: get PR
+      - name: get EE
         uses: actions/checkout@v3
         with:
-          fetch-depth: 1  # we only need this branch
+          fetch-depth: 1  # we only need the main branch
           token: ${{ secrets.DETERMINED_TOKEN }}
+          repository: ${{ EE_REPO }}
       - name: add EE repository remote and get default branch
         run: |
-          DEFAULT_BRANCH=$( gh api repos/${{ env.EE_REPO }} -q '.default_branch' )
-          git remote add EE github.com:${{ env.EE_REPO }}
-          git fetch EE
-          git checkout EE/$DEFAULT_BRANCH
+          git remote add OSS github.com:${{ env.OSS_REPO }}
+          git fetch OSS "${{ github.ref }}":proposed_base
```

Comment:
Where does proposed ref come from?

Reply:
That should be the ref name used in the local repo.  It's consumed in the rebase step.